### PR TITLE
Dlna/Chromecasts Module Improvement

### PR DESCRIPTION
**Pull request by team-pct**
DLNA / Chromecast Module Imrprovement

### CODE REVIEW

File: src/app/lib/device/chromecast.js [hunk 1]
```diff
@@ -57,25 +52,22 @@
...
-            });
-            this.device.on('status', function (status) {
+  				self.device.play(url, media, function (err, status) {
+  					if (err) {
+  						win.error('chromecast.play error: ', err);
+  					} else {
+  						win.info('Playing ' + url + ' on ' + self.get('name'));
+  						self.set('loadedMedia', status.media);
+  					}
+  				});
```
Comment (xaiki) [inline src/app/lib/device/chromecast.js]: indent is off.

File: src/app/lib/device/chromecast.js [hunk 2]
```diff
@@ -135,8 +127,8 @@
 
         updateStatus: function () {
             var self = this;
-
-            this.get('device').getStatus(function (err, status) {
+            console.log(this.get('player'));
```
Comment (xaiki) [inline src/app/lib/device/chromecast.js]: forgotten debug ?

File: src/app/lib/device/chromecast.js [hunk 3]
```diff
@@ -150,16 +142,17 @@
             }
             // If this is the active device, propagate the status event.
             if (collection.selected.id === this.id) {
+              console.log(status);
```
Comment (xaiki) [inline src/app/lib/device/chromecast.js]: forgotten debug ?

File: src/app/lib/device/chromecast.js [hunk 4]
```diff
@@ -150,16 +142,17 @@
...
         }
     });
 
-    var browser = new chromecast.Browser();
 
-    browser.on('deviceOn', function (device) {
-        collection.add(new Chromecast({
-            device: device
+chromecasts.update();
+chromecasts.on('update', function (player) {
```
Comment (xaiki) [inline src/app/lib/device/chromecast.js]: indent seems off

**Comment by xaiki** [comment]
looks very good ! thank you for your colllaboration, please fix the small styles issues i marked and we can merge this.

**Comment by team-pct** [comment]
Done

**Comment by xaiki** [comment]
thanks heaps !

**Comment by ZakariaRidouh** [comment]
Hey Great Job @team-pct!

**Pull request merged**